js/interactive-guides/table-of-contents.js

```diff
@@ -137,17 +156,25 @@ var tableofcontents = (function() {
         @param - `span` is the span of the step in the table of contents
         @param - `step` is the JSON containing information for the step
     */
-    var __addOnClickListener = function(listItem, step) {
+    var __addOnClickListener = function(listItem, step, dataToc, section) {
         var span = listItem.find('.tableOfContentsSpan');
         span.on("click", function(event){
             event.preventDefault();
             event.stopPropagation();
             __handleFirstStepContent(step);
-
-            //console.log("Clicked step: " + step.name);
             stepContent.createContents(step);
 
-            scrollToContent();
+            // If the listItem is for a subsection scroll to it after loading the step
+            var focusSection = $("[data-section='" + section + "']");
```

Comment:
Where is data-section set? Is it possible that it should be data-toc as set in createListItem?

Reply:
That's from the json, the guide creator is to add the data tag to the section of the page where it should scroll to.

Reply:
I was looking for the json changes but couldn't find it. Where the json changes dropped or to be dropped?